### DISCUSSION TITLE
Combine same selector class with '+' or something like that

### DISCUSSION THREAD

**Discussion by sarowarhosen03** [tailwindlabs/tailwindcss#12645] [Ideas]
I wish we can write tailwind class like
Instead of writing this `dark:bg-black dark:text-white` can we make it like this 
`dark:bg-black+text-white` ?
`hover:bg-bule+text-white`
That's how the duplication of state selector will reduce

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

**Reply by sarowarhosen03** [reply]
Oh nice to see. Thanks for your reference
